apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -1187,7 +1187,7 @@ async fn management_router<I: InfraMapProvider>(
     let route = get_path_without_prefix(PathBuf::from(req.uri().path()), path_prefix);
     let route = route.to_str().unwrap();
     let res = match (req.method(), route) {
-        (&hyper::Method::POST, "logs") if !is_prod => Ok(log_route(req).await),
```

Comment:
I remember why I added the `is_prod` filtering. Because we don't want to use std out in prod mode.